cdap-hbase-compat-0.94/src/main/java/co/cask/cdap/data2/transaction/queue/coprocessor/hbase94/HBaseQueueRegionObserver.java

```diff
@@ -53,7 +54,8 @@
   private static final Log LOG = LogFactory.getLog(HBaseQueueRegionObserver.class);
 
   private Configuration conf;
-  private byte[] configTableName;
+  private byte[] configTableNameBytes;
+  private String sysConfigTableName;
```

Comment:
is it more accurate to call this sysConfigTableNamePrefix?  Or is this actually the table name in hbase94?